### ISSUE TITLE
Crash on launch

### ISSUE THREAD

**Issue by baduhai** [TheWiseNoob/OpenMusicPlayer#6]
Just installed your player after seeing it on Reddit a couple of time, am excited to try it out. I installed it from the AUR package, and whenever I open it I get the following output:
```
find() setting name: output.sink



find() setting name: output.sink



find() setting name: playback.volume



find() setting name: playback.volume



find() setting name: replaygain.pre_amp



find() setting name: replaygain.pre_amp



find() setting name: replaygain.fallback_gain



find() setting name: replaygain.fallback_gain



find() setting name: replaygain.gain_type



find() setting name: replaygain.gain_type



find() setting name: playback.queue_saved



find() setting name: playback.queue_saved

terminate called after throwing an instance of 'Glib::FileError'
Aborted (core dumped)
```

**Comment by TheWiseNoob** [comment]
One of the icon images must not be getting installed properly. It's working for me, but it's probably because I already had the image from a previous install. I'll have a fix for it on Sunday.

**Comment by TheWiseNoob** [comment]
Sorry I didn't get to this yesterday. I just released version 0.0.3 as well as a fix for your problem today. Reinstall via omp-git on AUR. Please let me know if it works or not! I'm pretty sure it should. I tested it pretty thoroughly.

**Comment by baduhai** [comment]
It works! Thanks so much.
